### ISSUE TITLE
When to use includePolyfill: true?

### ISSUE THREAD

**Issue by mydea** [babel/ember-cli-babel#259]
It is not clear to me what the status on `includePolyfill: true` is. There are some outstanding issues/PRs from 2017:

https://github.com/babel/ember-cli-babel/issues/166
https://github.com/babel/ember-cli-babel/pull/142
https://github.com/babel/ember-cli-babel/pull/187

Then when looking at core-js, it says that babel-preset-env should take care of that, but not sure if that applies here (see https://github.com/zloirock/core-js/issues/188)?

Also, I'm not quite sure how this plays together with https://github.com/machty/ember-maybe-import-regenerator.

Basically, in what scenarios do I need to set `includePolyfill: true`? If we target fairly modern browsers (e.g. last 1 chrome, safari, firefox), what do we need to include? The polyfill? Or only ember-maybe-import-regenerator? Or neither of them? I could not find any definite information on this, and since the polyfill does add about 30KB gzipped to our total bundle size, it would be a fairly big benefit in being able to drop it. 

Maybe someone can help clear this up for me :)

**Comment by acorncom** [comment]
@mydea https://github.com/pzuraq/ember-cli-babel-polyfills is a recent addon that aims to help resolve your question ...

**Comment by bertdeblock** [comment]
Going to close this as polyfill support was removed in v8.
